devito/ir/clusters/algorithms.py

```diff
@@ -226,7 +226,7 @@ def callback(self, clusters, prefix):
 
         d = prefix[-1].dim
 
-        subiters = flatten([c.ispace.sub_iterators.get(d, []) for c in clusters])
+        subiters = flatten([c.ispace.sub_iterators[d] for c in clusters])
```

Comment:
So we sure `d` is always an existing key?

Reply:
In this case yes, as it is part of the cluster's dimensions and ispace

Reply:
What's the advantage/difference of get vs []?